src/snowflake/cli/plugins/nativeapp/commands.py

```diff
@@ -229,17 +230,44 @@ def app_teardown(
 @app.command("deploy", requires_connection=True)
 @with_project_definition("native_app")
 def app_deploy(
+    prune: Optional[bool] = typer.Option(
+        default=None,
+        help=f"""Whether to delete specified files from the stage if they don't exist locally. If set, the command deletes files that exist in the stage, but not in the local filesystem.""",
+    ),
+    recursive: Optional[bool] = typer.Option(
+        None,
```

Comment:
Personally I would use `False` as default for boolean value

Reply:
I needed it to be `None` here because there is the special flow of running `snow app deploy` with no arguments, in which case the default value is `True`